WooCommerce/Classes/System/WooConstants.swift

```diff
@@ -73,6 +73,10 @@ extension WooConstants {
         ///
         case helpCenter = "https://docs.woocommerce.com/document/woocommerce-ios/"
 
+        /// Help Center for "Enter your Store Address" screen
+        ///
+        case helpCenterForEnterStoreAddress = "https://woocommerce.com/document/android-ios-apps-login-help-faq/#section-1"
```

Comment:
nit: might be nice if the anchor name tells what the section is? like `https://woocommerce.com/document/android-ios-apps-login-help-faq/#enter-store-address` if it's easy to update the webpage. also, we can probably shorten the URL like from `https://woocommerce.com/document/android-ios-apps-login-help-faq` to `https://woocommerce.com/document/mobile-apps-login-faq`

Reply:
Thanks for the inputs. 🙏 

> nit: might be nice if the anchor name tells what the section is? like https://woocommerce.com/document/android-ios-apps-login-help-faq/#enter-store-address if it's easy to update the webpage.

💯  I thought the same, but I couldn't figure out how to change it. I thought that there is no way to do it. I will ask @hafizrahman  or someone else about this. 😄 

> we can probably shorten the URL like from https://woocommerce.com/document/android-ios-apps-login-help-faq to https://woocommerce.com/document/mobile-apps-login-faq

I think it was named like this on purpose for SEO reasons. I will check about this as well.

Reply:
If you have edit permission in the WP site, you can edit the anchor text in the Heading block settings > Advanced:

<img width="256" alt="Screen Shot 2022-08-24 at 10 34 22 AM" src="https://user-images.githubusercontent.com/1945542/186305160-1575eff0-ced9-4e85-aa63-078ff3be10d3.png">

Reply:
That worked. Thank you!

Reply:
Updated URL in https://github.com/woocommerce/woocommerce-ios/pull/7553/commits/24c27cde85342c2e5631f956e8550bd4984f50dd